dnsimple/dnsimple-support
# Deploy to Netlify

Fixes #491. This PR deploys this repository to Netlify automatically when the `master` branch is updated.

### Highlights
1. Removes `s3_website` gem and configuration files
2. Updates Travis config to no longer deploy. It runs the tests, and compile the pages.
3. Updates README.md
4. Adds canonical link to avoid SEO duplication.

### Verification steps
- Verify that the canonical header is correct when testing this branch locally
- Netlify has been already been configured to deploy the latest on `master`. Check a few pages here: dnsimple-support.netlify.com

### Post steps

- [ ] Configure DNS records to point to dnsimple-support.netlify.com
